openpype/hosts/max/plugins/publish/validate_loaded_plugin.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""Validator for Loaded Plugin."""
+from pyblish.api import ContextPlugin, ValidatorOrder
+from pymxs import runtime as rt
+
+from openpype.pipeline.publish import (
+    RepairContextAction,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+from openpype.hosts.max.api.lib import get_plugins
+
+
+class ValidateLoadedPlugin(OptionalPyblishPluginMixin,
+                           ContextPlugin):
+    """Validates if the specific plugin is loaded in 3ds max.
+    User can add the plugins they want to check through"""
```

Comment:
I believe I commented a tweak on this before too - but it isn't really up to the user, right? It's more like up to the Studio Admin via studio defined project settings?

Reply:
Maybe we can use "Studio Admin" instead of "user"